$OpenBSD: patch-src_modules_tl-util_jsm,v 1.2 2017/08/27 08:54:08 sthen Exp $

Force DataOutsideOfAppDir on OpenBSD since it will always be true
(turned on in ../browser/patches/patch-patch-xpcom_io_TorFileUtils_cpp)
Index: src/modules/tl-util.jsm
--- src/modules/tl-util.jsm.orig
+++ src/modules/tl-util.jsm
@@ -697,7 +697,7 @@ let TLUtilInternal =  // Private
   mStringBundle : null,
   mOS : "",
   // mIsUserDataOutsideOfAppDir is true when TorBrowser-Data is used.
-  mIsUserDataOutsideOfAppDir: undefined, // Boolean (cached; access via
+  mIsUserDataOutsideOfAppDir: true,      // Boolean (cached; access via
                                          //   this._isUserDataOutsideOfAppDir)
   mAppDir: null,        // nsIFile (cached; access via this._appDir)
   mDataDir: null,       // nsIFile (cached; access via this._dataDir)
